.github/workflows/nightly-tests.yml

```diff
@@ -66,12 +66,23 @@ jobs:
           vault-token: ${{ steps.vault-token-step.outputs.vault-token }}
           target: wsmtest
 
+      # The ArgoCD sync triggers the synchronization of the wsmtest cluster. Typically,
+      # the cluster is reset within a few seconds. However, it is possible for it to take
+      # minutes or hours. We have no way to check that the sync is complete. We sleep
+      # for a bit and just accept that in some crazy long cases, these tests will fail.
+      # The sync is bracketed by /version probes so we have the pre and post sync versions
+      # in the log.
```

Comment:
FWIW, I liked the idea of a separate workflow to promote the workspace manager. We could have separate reporting in Slack. But you'd have to align the cron schedules.